drivers/gpio/Kconfig.nrf5

```diff
@@ -26,7 +26,7 @@ config GPIO_NRF5_P0
 
 config GPIO_NRF5_P0_DEV_NAME
 	string "GPIO Port P0 Device Name"
-	depends on GPIO_NRF5_P0
+	depends on GPIO_NRF5_P0 && !HAS_DTS_GPIO
```

Comment:
This is just suggestion: how about grouping all properties dependent from !HAS_DTS_GPIO using `if !HAS_DTS_GPIO ` instead of adding dependency for each other. This will improve legibility of this file.

Reply:
Yes, I think I agree with you. I took the precedent from some existing Kconfig, which I've just looked again and it was from the I2C and ADC drivers. Looking at them again, I think this would apply to them also.

Will change.